consensus/polybft/consensus_runtime.go

```diff
@@ -126,6 +114,9 @@ type consensusRuntime struct {
 	// proposerCalculator is the object which manipulates with ProposerSnapshot
 	proposerCalculator *ProposerCalculator
 
+	// manager for state sync bridge transactions
+	stateSyncManager *StateSyncManager
```

Comment:
Personally I'm not very keen we are constantly checking `IsBridgeEnabled` on multiple places in consensus runtime...

Therefore, what do you think about introducing null object pattern:
1. abstract `StateSyncManager` to some interface (expose only necessary functions: `PostBlock`, `PostEpoch` etc. which looks very much like hook system)
2. have `NullStateSyncManager` implementation by default (if bridge not enabled) and no need to check constantly is bridge enabled. Only downside I see currently is that we would need to create those `request` objects in any way...

Reply:
Agreed. Will change it.

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/1067/commits/c693688956d77512f89b400048ba503057396c78